src/network/clientpackethandler.cpp

```diff
@@ -335,8 +335,7 @@ void Client::handleCommand_Inventory(NetworkPacket* pkt)
 
 	m_update_wielded_item = true;
 
-	delete m_inventory_from_server;
-	m_inventory_from_server = new Inventory(player->inventory);
+	m_inventory_from_server = std::make_unique<Inventory>(player->inventory);
 	m_inventory_from_server_age = 0.0;
```

Comment:
```suggestion
	m_inventory_from_server_age = 0.0f;
```

Reply:
Done (even if I'm not sure how much of "general cleanup" I can include in non related PRs).